chapters/annotations.tex

```diff
@@ -1192,7 +1192,10 @@ \section{Annotations for the Graphical User Interface}\doublelabel{annotations-f
 
 The annotations \lstinline!tab! and \lstinline!group! define the placement of
 the component or of variables in a dialog with optional tab and group
-specification. If \lstinline!enable = false!, the input field may
+specification, where the empty string (default) means tool-specific tab and group.
```

Comment:
I'd prefer keeping `tab` out of the specification here; it just seems too strange to have a default value for `tab`, and then define that an empty `group` makes this value go away.  I can't recall this kind of magic for any other annotation.

If we wanted something more clever for `tab`, the only reasonable way would be to treat it in the same way as `group`, with an empty string as default value, to be interpreted as freedom for the tool to choose.  However, I think `"General"` is general enough to actually be a useful default, meaning that a user actually knows what to expect when leaving `tab` at its default.  The main problem, as I see it, is the lack of symmetry between the handling of `group` and `tab`.

Reply:
I agree with that change. I just copied that part from the decision.

Reply:
OK.  Resolved.